stainless_frontend/tests/pass/phantom_data.rs

```diff
@@ -0,0 +1,22 @@
+extern crate stainless;
+
+use std::marker::PhantomData;
```

Comment:
What happens if we `use core::marker::PhantomData` here? Does it still work?

Reply:
Good question, hadn't thought about that. Indeed the test fails if we directly import from `core`.

Reply:
Otherwise, the DefId logic continues to work correctly, because the DefId seems to be independent of the `def_path_str`.
Hence, the question: should I add this to the PR?

```rust
@@ -22,13 +22,14 @@ fn should_ignore(item: &hir::Item<'_>) -> bool {
   match item.kind {
     ItemKind::ExternCrate(_) => {
       let name = item.ident.name.to_string();
-      name == "std" || name == "stainless"
+      name == "std" || name == "stainless" || name == "core"
     }
     ItemKind::Use(ref path, _) => {
       let path_str = pretty_path(path);
       path_str.contains("std::prelude")
         || path_str.starts_with("std::marker::PhantomData")
         || path_str.starts_with("stainless")
+        || path_str.starts_with("core")
     }
     // TODO: Quick fix to filter our synthetic functions
     // ItemKind::Fn(..) if !item.attrs.is_empty() => true,
```

Reply:
Ok great! No need to tackle this right but good to know :)